### PR TITLE
Update enable_rdp.rb

### PR DESCRIPTION
**Pull request by vysecurity**
Fixed some typos.

### CODE REVIEW

File: modules/post/windows/manage/enable_rdp.rb [hunk 1]
```diff
@@ -31,8 +31,8 @@ def initialize(info={})
         OptString.new('USERNAME', [ false, 'The username of the user to create.' ]),
         OptString.new('PASSWORD', [ false, 'Password for the user created.' ]),
         OptBool.new(  'ENABLE',   [ false, 'Enable the RDP Service and Firewall Exception.', true]),
-        OptBool.new(  'FORDWARD', [ false, 'Forward remote port 3389 to local Port.', false]),
```
Comment (wvu-r7) [inline modules/post/windows/manage/enable_rdp.rb]: Wow, haha.

Comment (vysec) [inline modules/post/windows/manage/enable_rdp.rb]: :dancer:

**Comment by vysec** [comment]
Hmm the comment you quoted has still an incorrect FORDWARD? Though it's fixed in my PR. Not sure what's wrong. 

V

**Comment by wvu-r7** [comment]
I was commenting on the deletion. :)

**Comment by sempervictus** [comment]
Another testament to tab completion making asses of us all. Good catch.

**Comment by vysec** [comment]
@sempervictus Very nice, now I know the root cause!

V

**Pull request merged**